queries/vm_backup.kusto

```diff
@@ -0,0 +1,24 @@
+let VMs = MonitoringResources_CL 
+| where TimeGenerated > ago(13h)
+| where type_s == "microsoft.compute/virtualmachines"
+| summarize take_anyif(TimeGenerated, isnotempty(properties_extended_instanceView_osName_s) ) by name_s, tags_managedby_s, properties_storageProfile_osDisk_osType_s, ResourceId
+| project vmResourceID = ResourceId, name = tolower(name_s), managedby = tags_managedby_s, osType = properties_storageProfile_osDisk_osType_s;
```

Comment:
```suggestion
| summarize take_anyif(TimeGenerated, isnotempty(properties_extended_instanceView_osName_s) ) by name = tolower(name_s), managedby = tags_managedby_s, osType = properties_storageProfile_osDisk_osType_s, vmResourceID = ResourceId
```
You should be able to avoid projecting twice. Furthermore you could already Filter for only q.beyond managed before to reduce the amount of data.

Reply:
Yeah makes sense. I also just noticed that the take_anyif is a left over from a previous issue where we used a column that didn't always contain data. This is fixed now with properties_extended_instanceView_osName_s. So I will replace it with a normal arg_max on TimeGenerated to make it more readable